doc/Changes.md

```diff
@@ -0,0 +1,6344 @@
+# GAP - Changes from Earlier Versions
+
+Release 4.11.0, ??-???-2020
+
+**The GAP Group**  
+Email: <support@gap-system.org>  
+Homepage: <https://www.gap-system.org>
+
+### Copyright
+
+Copyright © (1987-2020) for the core part of the **GAP** system by the
```

Comment:
Do we really wan to write `**GAP**` throughout? I'd suggest we change all of these to just `GAP` via search&replace.

Reply:
Agree